<launch>
    <!-- ochang -->
    <node name="map_server" pkg="map_server" type="map_server" args="$(find localizer)/map/ochang_grid.yaml" >
    </node>
    <!-- k-city -->
    <!-- <node name="map_server" pkg="map_server" type="map_server" args="$(find localizer)/map/global_map.yaml" >
    </node> -->

    <node pkg="localizer" name ="localizer" type="localizer" output="screen">
        <!-- gnss_localizer -->
        <param name="gps_velocity_estimate"         value="true"      type="bool"/>
        <param name="gps_subscribe_topic_name"      value="/ebimu_"  type="string"/>
        <param name="gps_bestvel_subscribe_topic_name"  value="/bestvel"  type="string"/>
        <param name="gps_bestpos_subscribe_topic_name"  value="/bestpos"  type="string"/>
        <param name="initialpose_topoc_name"  value="/initialpose"  type="string"/>
        <param name="covariance_sample_num"         value="25"         type="int"/>

        <!-- Imu orientation -->
        <param name="imu_subscribe_topic_name" value ="/vectornav/IMU" type="string"/>
        <!-- Encoder -->
        <param name="Velocity_topic_name"         value="/MSG_CON/Rx_Vel"         type="string"/>

        <!--ochang-->
        <param name="utmOffset_x"    value="361001.412425"  type="double"/>
        <param name="utmOffset_y"    value="4065821.07176"  type="double"/>

        <!-- k-city -->
        <!-- <param name="utmOffset_x"    value="302533.174487"  type="double"/>
        <param name="utmOffset_y"    value="4124215.34631"  type="double"/> -->
    </node>
</launch>
